lib/IMAP/MailboxSync.php

```diff
@@ -31,6 +31,7 @@
 use Horde_Imap_Client_Namespace_List;
 use OCA\Mail\Events\MailboxesSynchronizedEvent;
 use OCA\Mail\Exception\ServiceException;
+use OCP\DB\Exception;
```

Comment:
Psalm: Does not exist for Nextcloud 20 :thinking:

Reply:
Then let's do without the exception. That wasn't my focus anyway :)